src/portfolio.js

```diff
@@ -185,8 +189,9 @@ To know how to get github key look at readme.md */
 
 const openSource = {
   githubConvertedToken: process.env.REACT_APP_GITHUB_TOKEN,
-  githubUserName: "saadpasta", // Change to your github username to view your profile in Contact Section.
-  showGithubProfile :"true" // Set true or false to show Contact profile using Github, defaults to false
+  githubUserName: "Your Github Username", // Change to your github username to view your profile in Contact Section.
+  showGithubProfile : false,  // Set true or false to show Contact profile using Github, defaults to false
+  display: true // Set true to display this section, defaults to false
```

Comment:
set `showGithubProfile `to true and add `githubUserName` **saadpasta**

Reply:
ready :+1: